### ISSUE TITLE
No horizonal spacing between edittext and widget

### ISSUE THREAD

**Issue by wave9d** [Immediate-Mode-UI/Nuklear#539]
So I have an edit text component and a line drawn with a widget.
I want the line to go right under the edit text component without ANY spacing.
Is this achivable with what nuklear offers at the moment ? 

This is what I managed to do so far, but I want no spacing between the two.
![image](https://user-images.githubusercontent.com/106614552/212434515-070c3147-51c8-4879-a4fb-729b0f9497c5.png)

**Comment by dumblob** [comment]
Without looking at the source/API, if the horiz. bar/line does not allow the padding/spacing to be changed (either by `ctx.blabla` or directly in the function interface), then you can easily copy & paste the function into your code, rename it and change the padding to your own values. Nuklear is really hackable.

A better option would be to add the padding setting to `ctx.blabla` (if it is not yet supported) and make a PR to this repo :wink:.

**Comment by wave9d** [comment]
Sorry, I meant vertical spacing.

**Comment by dumblob** [comment]
Yes, I understood (based on your picture). My reply above thus holds :wink:.
